src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -1,8 +1,44 @@
 /**
  * A sample SIMID ad that shows a map of nearby locations
  */
+const AdParamKeys = {
+  BUTTON_LABEL: 'buttonLabel',
+  SEARCH_QUERY: 'searchQuery',
+  MARKER: 'marker',
+}
+const DEFAULT_BUTTON_LABEL = "Location";
+
 class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    const adParams = this.creativeData.adParameters;
+    const buttonLabel = JSON.parse(adParams)[AdParamKeys.BUTTON_LABEL]; 
```

Comment:
Does this work? buttonLabel is const, but then you reassign it below, I think it should be let

Reply:
didn't mean to change this variable